app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The post belongs to the author, kindly consider implementing this so the relations are setup correctly
e.g ``belongs_to :author, class_name: 'User'``